features/support/run_helpers.rb

```diff
@@ -55,9 +55,8 @@ def output_of command
 
 def print_result result
   puts ''
-  puts "#{result.location}$ #{result.command}"
-  puts "#{result.out}"
-  puts ''
+  puts %(#{result.location}$ #{result.command}
+         #{result.out}).gsub(/^/, '@ ')
```

Comment:
I'll add it back. Since `puts` adds a trailing newline, I thought this was unnecessary. I'll just keep it as-is for now